geomstats/backend/tensorflow/__init__.py

```diff
@@ -23,6 +23,10 @@ def logical_or(x, y):
     return tf.logical_or(x, y)
 
 
+def logical_and(x, y):
```

Comment:
For the functions that have exactly the same syntax, I wonder if we could the same code as here:
https://github.com/geomstats/geomstats/blob/446f0f0a754b4d0ff7660fb240f4822e8eb673d3/geomstats/backend/numpy/__init__.py#L4

Eg:

from tensorflow import (
    logical_and,
...)

Reply:
update on this?